src/Compilers/CSharp/Test/Emit/Emit/EntryPointTests.cs

```diff
@@ -1642,5 +1642,41 @@ class Program
                 // error CS5001: Program does not contain a static 'Main' method suitable for an entry point
                 Diagnostic(ErrorCode.ERR_NoEntryPoint));
         }
+
+        [Fact]
+        public void AbstractEntryPointWithoutImplementation()
```

Comment:
>AbstractEntryPointWithoutImplementation [](http://example.com/codeflow?start=20&length=39)

I suggest moving the tests to src\Compilers\CSharp\Test\Symbol\Symbols\StaticAbstractMembersInInterfacesTests.cs and following the pattern used in that file for target framework and expected output in tests.